docs/conf.py

```diff
@@ -300,7 +300,8 @@ def source_replace(app, docname, source):
 
 # Dict of replacements.
 source_replacements = {
-    "{PLONE_BACKEND_VERSION}": "6.0.0",
+    "{PLONE_BACKEND_MINOR_VERSION}": "6.0",
+    "{PLONE_BACKEND_VERSION}": "6.0.1",
```

Comment:
I don't see the point in renaming the variable. Let's keep it short.

```suggestion
    "{PLONE_BACKEND_VERSION}": "6.0",
```

Reply:
Buildout needs the PATCH version. See comment above.

Reply:
OK, in that case, let's be explicit with the name.

```suggestion
    "{PLONE_BACKEND_PATCH_VERSION}": "6.0.1",
```

Reply:
However, where do we document the use of buildout? I thought [we made the decision not to do that in Plone 6](https://6.docs.plone.org/install/install-from-packages.html#install-plone-6)?

Reply:
No buildout. I should not have mentioned it.
It's the installation with pip that uses the constraints of the selected Plone version. The url is https://dist.plone.org/release/{PLONE_BACKEND_VERSION}/constraints.txt with the complete PATCH version

Reply:
I think this is the only place that needs that distinction:

https://github.com/plone/documentation/blame/6-dev/docs/install/manage-add-ons-packages.md#L64

Reply:
https://dist.plone.org/release/6.0-latest/constraints.txt is available. But, in the context of a project that's going to be maintained over time the best practice is to pin to a specific version, so that your CI doesn't break until you've submitted a PR to explicitly update to the new patch (so that if it breaks, you know why)

Reply:
> https://dist.plone.org/release/6.0-latest/constraints.txt is available. But, in the context of a project that's going to be maintained over time the best practice is to pin to a specific version, so that your CI doesn't break until you've submitted a PR to explicitly update to the new patch (so that if it breaks, you know why)

I'm not sure if I understand. The documentation recommends to pin the version.

Reply:
@ksuess Sorry, I also ended up recommending to pin the version, but I was not very clear.